client/packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -86,7 +103,15 @@ export const DetailView: FC = () => {
             />
           )}
 
-          <Toolbar />
+          {returnsIsOpen && (
+            <InboundReturnEditModal
+              isOpen={returnsIsOpen}
+              onClose={onCloseReturns}
+              stockLineIds={stockLineIds || []}
+            />
+          )}
+
+          <Toolbar onReturnLines={onReturn} />
```

Comment:
How do you feel about placing onReturn logic in toolbar vs here ?

Reply:
Are you meaning you think it would better belong in the Toolbar? I preferred trying to remove responsibility from the Toolbar, but curious your thoughts?